### ISSUE TITLE
Directional symbols identified by Talkback as “Button, disabled”

### ISSUE THREAD

**Issue by shaunan** [StatCan/StatcanPublicationUAT#592]
On the “Privacy notice” page, there are left and right directional symbols that may be intended for navigation but are presently not active and appear grey in colour. These are both identified by Talkback as “Button, disabled”. If the user navigated to another page linked to from the “Privacy notice” page, then the new page is displayed and the left directional symbol now appears black in colour, and is identified as “Button, double tap to activate”. A screen reader user can not identify the purpose of these buttons, which function similarly to “Previous” and “Next” functions.

Provide a label for these navigational buttons to declare their function to assistive technology users. Note: all scenarios for when these two buttons appear are documented by Rajat on the Confluence page.

![GitHub5](https://github.com/StatCan/StatcanPublicationUAT/assets/115822055/8bb7f07f-80bc-48b1-8fa0-8c7f61b4a1e2)
![GitHub5-1](https://github.com/StatCan/StatcanPublicationUAT/assets/115822055/64f691b2-51ce-4e73-91f5-812ce032d73f)
